modules/nf-core/taxonkit/name2taxid/meta.yml

```
@@ -0,0 +1,53 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "taxonkit_name2taxid"
+description: Convert taxon names to TaxIds
+keywords:
+  - taxonomy
+  - taxIds
```

```suggestion
  - taxids
```
I thin kthey are meant to be all lower case